docs/faq.md

```diff
@@ -201,7 +201,7 @@ argocd ... --insecure
 ## I have configured Dex via `dex.config` in `argocd-cm`, it still says Dex is unconfigured. Why?
 
 Most likely you forgot to set the `url` in `argocd-cm` to point to your ArgoCD as well. See also
-[the docs](/operator-manual/user-management/#2-configure-argo-cd-for-sso).
+[the docs](./operator-manual/user-management/#2-configure-argo-cd-for-sso).
```

Comment:
Looks like this is still broken in [the test build](https://argo-cd--10744.org.readthedocs.build/en/10744/faq/). I think this would work:

```suggestion
[the docs](./operator-manual/user-management/index.md#2-configure-argo-cd-for-sso).
```

Reply:
Just pushed requested change, i wonder is there any application that could help prevent any documentation error like parsing links on repository and test build?